docs/best-practices/serverless/azure.md

```diff
@@ -21,131 +21,133 @@ To get you up and started, we've put together an example repository to be used t
-        
+
+       ![Screen Shot 2022-09-13 at 3.49.15 PM.png](/Screen_Shot_2022-09-13_at_3.49.15_PM.png)
+
     4. Save the configuration and deploy again in VSCode
-5. Once deployed, you should see this message
-    
+
+5.  Once deployed, you should see this message
+
     ![Screen Shot 2022-09-13 at 3.52.40 PM.png](/Screen_Shot_2022-09-13_at_3.52.40_PM.png)
-    
-6. Go back to your Azure Portal and set the environment variable `SERVER_KEY` in your function app configuration and save. [You can learn more about SDK keys and where to find them here.](/essentials/keys)
-    
+
+6.  Go back to your Azure Portal and set the environment variable `SERVER_KEY` in your function app configuration and save. [You can learn more about SDK keys and where to find them here.](/essentials/keys)
+
     ![Screen Shot 2022-09-13 at 3.55.40 PM.png](/Screen_Shot_2022-09-13_at_3.55.40_PM.png)
-    
+
 7.  Go to your Function (i.e. HttpTrigger1), and “Test/Run” it
-    
+
     ![Screen Shot 2022-09-13 at 4.02.08 PM.png](/Screen_Shot_2022-09-13_at_4.02.08_PM.png)
-    
-8. You should see the data you have set earlier in DevCycle dashboard!
-    
+
+8.  You should see the data you have set earlier in DevCycle dashboard!
+
     ![Screen Shot 2022-09-13 at 4.07.43 PM.png](/Screen_Shot_2022-09-13_at_4.07.43_PM.png)
-    
-9. You can play around the feature flags  we have set earlier and run the function again to see the result changed
+
+9.  You can play around the feature flags we have set earlier and run the function again to see the result changed
+
     1. For example, let’s “Serve” the “Variation Off” for `campaign-switch` and “Save”
-        
-        ![Screen Shot 2022-09-13 at 12.06.57 PM.png](/Screen_Shot_2022-09-13_at_12.06.57_PM.png)
-        
-    2.   Run your function again
-    3.  You should now see the following result
-        
-        ![Screen Shot 2022-09-13 at 4.09.54 PM.png](/Screen_Shot_2022-09-13_at_4.09.54_PM.png)
-    
+
+       ![Screen Shot 2022-09-13 at 12.06.57 PM.png](/Screen_Shot_2022-09-13_at_12.06.57_PM.png)
+
+    2. Run your function again
+    3. You should now see the following result
+
+       ![Screen Shot 2022-09-13 at 4.09.54 PM.png](/Screen_Shot_2022-09-13_at_4.09.54_PM.png)
+
 ### Enable EdgeDB (NodeJS SDK Version)
 
 [First, you need to enable EdgeDB in the DevCycle Dashboard](/extras/edgedb)
 
-With the [DevCycle NodeJs SDK ](/sdk/server-side-sdks/node) we use the `enableCloudBucketing` and `enableEdgeDB` options to enable EdgeDB usage. ([SDK initialization option details: ](/sdk/server-side-sdks/node/node-gettingstarted#initialization-options))
+With the [DevCycle NodeJs SDK](/sdk/server-side-sdks/node) we use the `enableCloudBucketing` and `enableEdgeDB` options to enable EdgeDB usage. ([SDK initialization option details:](/sdk/server-side-sdks/node/node-gettingstarted#initialization-options))
 
 ```jsx
 const devcycleClient = DVC.initialize(serverKey, {
```

Comment:
should update all the calls to not use the `DVC` acronym anymore since we're trying to use the full name `devcycle` in sample code

Reply:
will change in the example apps too